### PR TITLE
Docs gen - the latest

### PR DESCRIPTION
**Pull request by jamesaimonetti**

### CODE REVIEW

File: applications/crossbar/doc/ref/conferences.md [hunk 1]
```diff
@@ -89,6 +89,19 @@ curl -v -X POST \
...
+> PUT /v2/accounts/{ACCOUNT_ID}/conferences/{CONFERENCE_ID}
+
+```curl
+curl -v -X PUT \
+    -d '{"data": {"action": {CONFERENCE_ACTION}}}' \
+    -H "X-Auth-Token: {AUTH_TOKEN}" \
+    http://{SERVER}:8000/v2/accounts/{ACCOUNT_ID}/conferences/{CONFERENCE_ID}
+```
+
+CONFERENCE_ACTION: lock, unlock
```
Comment (jamesaimonetti) [inline applications/crossbar/doc/ref/conferences.md]: @jamhed These manual updates don't go in `crossbar/doc/ref` versions - these are auto-generated by the tool and your changes will be lost. What I meant was, you should take the diff set here in the `ref` version and copy over the relevant changes to the `crossbar/doc` version of `conferences.md`. Does that make sense? Sorry for the confusion.

Comment (jamhed) [inline applications/crossbar/doc/ref/conferences.md]: @jamesaimonetti, it does.

**Comment by k-anderson** [comment]
@jamesaimonetti where you waiting for further changes, or is the ready?

**Comment by jamesaimonetti** [comment]
@k-anderson waiting for @jamhed to move his changes from `doc/ref/` to `doc/` version.

**Pull request merged**